wemake_python_styleguide/visitors/ast/function_empty_lines.py

```diff
@@ -73,7 +74,12 @@ def _is_function_end(
         token: tokenize.TokenInfo,
         function_tokens_exists: bool,
         function_start_column: int,
+        function_start_line: int,
     ) -> bool:
+        is_elipsis = token.string == '...'
+        is_elipsis_end = is_elipsis and token.start[0] == function_start_line
```

Comment:
what about 

```python
def some(
   long_arg1: long_annotation1,
   long_arg2: long_annotation2,
) -> None: ...
```

?

Reply:
@sobolevn I'm add test for this case. Here `_is_function_end` detect end of function by `DEDENT` token

Reply:
Maybe this is a bug in the tokenizer. Maybe I can send PR to cpython?

Reply:
Let's first try to create a minimal repro example :)